lib/src/wave_dumper.dart

```diff
@@ -127,7 +128,7 @@ class WaveDumper {
   $dateString
 \$end
 \$version
-  ROHD
+  ROHD v${Config.version}
```

Comment:
We could add a test to confirm the version shows up in generated VCDs?

Reply:
Seems like dart do not support optional parameters with named parameter.

https://github.com/dart-lang/language/issues/1076